### PR TITLE
Feature/12 controller adminservice

### PR DESCRIPTION
**Pull request by thejaneshin**
Requesting pull request to merge feature/12-controller-adminservice to development.

### CODE REVIEW

File: src/main/resources/log4j.properties [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+log4j.appender.stdout.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n
+
+#Direct log message to file
+log4j.appender.file=org.apache.log4j.FileAppender
+log4j.appender.file.File=log.txt
+log4j.appender.file.layout=org.apache.log4j.PatternLayout
+log4j.appender.file.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n
+
+#log4j.appender.file.File=log.html
```
Comment (jurczakn) [inline src/main/resources/log4j.properties]: remove unsued logger settings

File: src/test/java/com/revature/test/AdminServiceImplIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+  void testDeleteNonExistingAdmin() {
+    assertThrows(DeleteNonexistentException.class, () -> {
+      adminServiceImpl.deleteAdmin(nonExistingAdmin);
+    });
+  }
+
+  @Test
+  @Sql("admin-script.sql")
+  void testUpdateExistingAdmin() {
+    System.out.println(
```
Comment (jurczakn) [inline src/test/java/com/revature/test/AdminServiceImplIntegrationTest.java]: why print line

**Review by jurczakn — CHANGES_REQUESTED** [review]
Log Warn inside of catch blocks

File: src/test/java/com/revature/test/AdminServiceImplIntegrationTest.java (hunk 2 above)
Comment (Astralvix) [reply src/test/java/com/revature/test/AdminServiceImplIntegrationTest.java]: all set

File: src/main/resources/log4j.properties (hunk 1 above)
Comment (Astralvix) [reply src/main/resources/log4j.properties]: all set

**Review by jurczakn — APPROVED** [review]
good

**Pull request merged**